rfhub2/db/migrate.py

```diff
@@ -18,10 +20,15 @@ def has_revision(connection: Connection) -> bool:
 
 
 def migrate_db(engine: Engine) -> None:
-    alembic_cfg = Config("rfhub2/alembic.ini")
+    alembic_cfg_path = Path(__file__).resolve().parent.parent.parent / "rfhub2" / "alembic.ini"
+    alembic_cfg = Config(alembic_cfg_path)
     with engine.begin() as connection:
+        cwd = getcwd()
+        chdir(Path(__file__).resolve().parent.parent / "alembic" / "versions")
```

Comment:
This had to be done so hat app can find alembic.ini from any directory it's invoked.